ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -53,6 +72,8 @@ def __init__(self, tracer=None, config=None, service=None):
         self._tags = ci.tags(cwd=_get_git_repo())  # type: Dict[str, str]
         self._service = service
         self._codeowners = None
+        self._app_key = os.getenv("DD_APP_KEY")
+        self._code_coverage_enabled_by_api, self._test_skipping_enabled_by_api = self._check_enabled_features()
```

Comment:
Does this variable naming make more sense?
```suggestion
        self._code_coverage_enabled, self._test_skipping_enabled = self._check_enabled_features()
```

Reply:
I think the explicitness about the source of the enablement is important in this case, because there will also be environment variables that enable these features.